### ISSUE TITLE
[react-native-windows - 0.57.0-rc.4] - react-native windows - doesn't create windows folder with UWP application

### ISSUE THREAD

**Issue by a-tarasyuk** [Microsoft/react-native-windows#2056]
### Environment

1. `react-native -v`: 0.57.8
2. `npm ls rnpm-plugin-windows`: 0.2.8 
3. `npm ls react-native-windows`: **0.57.0-rc.4**
4. `node -v`: v8.15.0
5. `npm -v`: 6.4.1

### Steps to Reproduce

1. npm i react-native-cli -g
2. react-native init AppName
3. npm i rnpm-plugin-windows
4. react-native windows

### Expected Behavior

Should be created windows folder with UWP application

### Actual Behavior

```shell
Reading application name from package.json...
Reading react-native version from node_modules...
Checking for react-native-windows version matching 0.57.*...
Installing react-native-windows@0.57.0-rc.4...
Generating self-signed certificate...
Using Default Certificate. Use Visual Studio to renew it.
new windows/AppName/AppName_TemporaryKey.pfx
ENOENT: no such file or directory, open '/home/AppName/windows/AppName/AppName_TemporaryKey.pfx'
```

**Comment by rozele** [comment]
@a-tarasyuk - what environment are you running the command from? Are you generating the react-native-windows app from a Mac? Usually the CLI tool will generate a certificate instead of using the default certificate if running from Windows.

**Comment by a-tarasyuk** [comment]
@rozele Linux and Mac. Basically, the issue is simple, just need to create windows folder (in the previous version yo fs could create folders reclusive) before generating/copy default certificate, because certificate has to be copied to folders which do not exists. Previous version, which was based on yo API allowed use cli on any platform. What do you think does that make sense?

**Comment by rozele** [comment]
Definitely, I'll look at the PR. Thanks @a-tarasyuk.

**Comment by a-tarasyuk** [comment]
@rozele Thanks :+1:

**Comment by rozele** [comment]
FYI, PR merged and published in 0.57.0-rc.5.

**Comment by a-tarasyuk** [comment]
@rozele Thanks
